app/features/modelling/galaxy-builder/index.js

```diff
@@ -127,25 +134,26 @@ class GalaxyBuilderModel extends baseModel {
           }
         )
       );
-
     if (this.convolvePSF) {
       this.convolvePSF({
         texture: this.state.pixels
       });
       this.state.pixels({ copy: true });
     }
+    let message = '';
     if (this.state.shouldCompareToImage) {
       this.calculateDifference({
         texture: this.state.pixels,
         imageTexture: this.imageData
       });
       this.state.pixels({ copy: true });
-      this.eventHandlers.setScore(this.getScore());
+      message += `Difference Image - Score: ${this.getScore()}`;
```

Comment:
Now that we have translations, will this component need to be localised?

Reply:
Here's Galaxy Builder in Dutch: https://www.zooniverse.org/projects/tingard/galaxy-builder?language=nl

Localised text is stored in `/app/locales` then translated using `counterpart` or `react-translate-component`.  Here's an example that uses a templated string (and allows HTML content): https://github.com/zooniverse/Panoptes-Front-End/blob/1171e88d228a2ffdca02747886da2b32d37a6e8b/app/pages/project/home/talk-status.jsx#L33-L40

If Galaxy Builder is only ever going to be in English, then there's no pressing need to translate it.

Reply:
I think the lifetime of the project is such that the text can be left in English. If needed I can work more on translating it post-launch and after my placement.

Reply:
How would you go about making a non-english project (i.e. about page etc...)? Thinking more for galaxy zoo than builder.

Reply:
We're testing out a translations editor at https://pandora.zooniverse.org. It shows you existing text (created by the project owner) and lets you create new versions for each resource (project, workflow, tutorial, field guide, project page.)